client/components/Email/render.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { ServerStyleSheet, StyleSheetManager } from 'styled-components';
+import juice from 'juice';
+import beautify from 'js-beautify';
+
+import { reset, globals } from './styles';
+
+const inlineStylesWithMarkup = (EmailMarkup: React.ReactNode, extraStyles: string) => {
+	const stylesheet = new ServerStyleSheet();
+	const renderedStringFromEmailMarkup = ReactDOMServer.renderToStaticMarkup(
+		<StyleSheetManager sheet={stylesheet.instance}>{EmailMarkup}</StyleSheetManager>,
+	);
+	const basicMarkup = beautify.html(renderedStringFromEmailMarkup);
```

Comment:
My sense is that we'd want to minify rather than beautify this HTML before sending it to people. I can see why this is useful for testing, though.

Reply:
When scoping this project out, I encountered someone saying that email clients don't like minified code, and that explicit beautification helps them get along. I shall probe further on the beautify/minify question.